plugins/scaffolder/src/plugin.ts

```diff
@@ -83,13 +103,23 @@ export const RepoUrlPickerFieldExtension = scaffolderPlugin.provide(
   }),
 );
 
+/**
+ * A field extensions for picking users and groups out of the Catalog.
```

Comment:
```suggestion
 * A field extension for picking users and groups out of the Catalog.
```

Signed-off-by: Fredrik Adelöw <freben@gmail.com>